core/src/main/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotService.scala

```diff
@@ -56,50 +57,62 @@ class SnapshotService(protected val ctx: RawlsRequestContext,
     with WorkspaceSupport
     with LazyLogging {
 
-  def createSnapshot(workspaceName: WorkspaceName,
-                     snapshotIdentifiers: NamedDataRepoSnapshot
+  // Finds a workspace using the workspaceId then calls the createSnapshot method
+  def createSnapshotByWorkspaceId(workspaceId: String,
```

Comment:
Scala apparently can't handle method signatures that are differ by parameter type (and name of the parameter) so I had to give the methods specific names.